### DISCUSSION TITLE
What does the fox say?

### DISCUSSION THREAD

**Discussion by ddh** [Schweinepriester/github-profile-achievements#90] [Q&A] [answered]
I'm having a bit of Galaxy Brain here and I can't seem to remember what the fox says. Anyone know?

**Comment by GabrielGomesFer** [comment, marked as answer]
"Ring-ding-ding-ding-dingeringeding!
Gering-ding-ding-ding-dingeringeding!
Gering-ding-ding-ding-dingeringeding!"

That's what the fox says
